src/AsyncGenerator.yml

```diff
@@ -119,6 +120,8 @@
     - name: GetFieldValue
     - name: IsDBNull
     - name: WriteLine
+    ignoreAsyncCounterparts:
+    - rule: Obsolete
```

Comment:
New from 0.7.0. From now on we want to generate async counterparts for obsolete members, but do not want to call them. Also, we would want to add obsolete members by hands where they ceased to be generated.